userspace/libsinsp/examples/test.cpp

```diff
@@ -38,254 +48,347 @@ static sinsp_evt_formatter* default_formatter = nullptr;
 static sinsp_evt_formatter* process_formatter = nullptr;
 static sinsp_evt_formatter* net_formatter = nullptr;
 
-// Functions used for dumping to stdout
-void plaintext_dump(sinsp& inspector);
-void json_dump(sinsp& inspector);
-
 static void sigint_handler(int signum)
 {
-    g_interrupted = true;
+	g_interrupted = true;
 }
 
 static void usage()
 {
-    string usage = R"(Usage: sinsp-example [options]
+	string usage = R"(Usage: sinsp-example [options]
 
 Options:
-  -h, --help                    Print this page
-  -f <filter>                   Filter string for events (see https://falco.org/docs/rules/supported-fields/ for supported fields)
-  -j, --json                    Use JSON as the output format
-  -a, --all-threads             Output information about all threads, not just the main one
+  -h, --help                                 Print this page.
+  -f <filter>, --filter <filter>             Filter string for events (see https://falco.org/docs/rules/supported-fields/ for supported fields).
+  -j, --json                                 Use JSON as the output format.
+  -a, --all-threads                          Output information about all threads, not just the main one.
+  -e <engine_name>, --engine <engine_name>   Specify the engine name to open possible values are: "kmod", "bpf", "udig", "nodriver", "savefile", "gvisor", "modern_bpf", "test_input". 
```

Comment:
Could we have the same approach for sinsp and scap-open examples? https://github.com/falcosecurity/libs/pull/521 - either approach will work.

Reply:
Not sure here if this would break @Molter73 tests...